File: python/1/1.py
def parse_line(line: str) -> int:
    line = line.strip()
    digits = [int(sym) for sym in line if sym.isdigit()]
    return digits[0] * 10 + digits[-1]


cumsum = 0
with open("1/input1.txt") as f:
    for line in f.readlines():
        cumsum += parse_line(line)
print(cumsum)

File: julia/1/1.jl
open("julia/1/input1.txt") do f
    sum_n = 0
    for line in eachline(f)
        list_digits = Vector{Int}()
        for char in line
            if isdigit(char)
                push!(list_digits, parse(Int, char))
            end
        end
        sum_n += list_digits[1] * 10 + list_digits[end]
    end
    println(sum_n)
end